README.md

```diff
@@ -1 +1,56 @@
-# dol-whd-14c
\ No newline at end of file
+# Department of Labor - Wage and Hour - Section 14c
+
+[![Build status](https://ci.appveyor.com/api/projects/status/gmq5jhbib0ug3rat?svg=true)](https://ci.appveyor.com/project/DOL-WHD-Section14c/master) 
+[![Code Climate](https://codeclimate.com/github/AppliedIS/dol-whd-14c/badges/gpa.svg)](https://codeclimate.com/github/AppliedIS/dol-whd-14c) 
+[![Javascript Coverage Status](https://coveralls.io/repos/github/AppliedIS/dol-whd-14c/badge.svg?branch=master)](https://coveralls.io/github/AppliedIS/dol-whd-14c?branch=master)
+[![DotNet Coverage Status](http://dol-whd-section14c-coverage-dev.azurewebsites.net/badge_combined.svg)](http://dol-whd-section14c-coverage-dev.azurewebsites.net/)
+[![dependencies Status](https://david-dm.org/AppliedIS/dol-whd-14c/status.svg?path=DOL.WHD.Section14c.Web)](https://david-dm.org/AppliedIS/dol-whd-14c?path=DOL.WHD.Section14c.Web)
+
+## Table of contents
+
+ - [Project Management](#project-management)
+ - [Application](#application)
+  - [User Interface](#user-interface)
+  - [REST Services](#rest-services)
+ - [Installing](#installing)
+ - [Deploying](#deploying)
+
+## Project Management
+
+Install [Zenhub](https://www.zenhub.com/) extension
+
+## Application
+
+### User Interface
+
+See [DOL.WHD.Section14c.Web Readme](DOL.WHD.Section14c.Web/Readme.md)
+
+| Project | Description     
+| --- | --- 
+DOL.WHD.Section14c.Web | AngularJS front-end
+
+### REST Services
+```
+DOL.WHD.Section14c.sln
+```
+| Project | Description     
+| --- | --- 
+DOL.WHD.Section14c.Api | ASP.Net WebAPI REST Services
+DOL.WHD.Section14c.Business | .Net Class Library for business services
+DOL.WHD.Section14c.Business.Test | MSTest Unit Tests
+DOL.WHD.Section14c.Common | .Net Class Libraray for shared coded
+DOL.WHD.Section14c.Domain | .Net Class Libarary for entities
+DOL.WHD.Section14c.DataAccess | .Net Class Libarary for Entity Framework Context
+
+## Installing
+
+## Running
+
+## Deploying
+
+WebDeploy Packages
+
+- DOL.WHD.Section14c.Web.zip
+- DOL.WHD.Section14c.API.zip
+
+
```

Comment:
This is also okay for now, but it will need to be fleshed out further before the end.  18F should provide some clarity about what all must be here.